prow/config/tide_test.go

```diff
@@ -33,33 +34,100 @@ import (
 var testQuery = TideQuery{
 	Orgs:                   []string{"org"},
 	Repos:                  []string{"k/k", "k/t-i"},
+	ExcludedRepos:          []string{"org/repo"},
 	Labels:                 []string{labels.LGTM, labels.Approved},
 	MissingLabels:          []string{"foo"},
 	Author:                 "batman",
 	Milestone:              "milestone",
 	ReviewApprovedRequired: true,
 }
 
+var expectedQueryComponents = []string{
+	"is:pr",
+	"state:open",
+	"archived:false",
+	"label:\"lgtm\"",
+	"label:\"approved\"",
+	"-label:\"foo\"",
+	"author:\"batman\"",
+	"milestone:\"milestone\"",
+	"review:approved",
+}
+
 func TestTideQuery(t *testing.T) {
 	q := " " + testQuery.Query() + " "
```

Comment:
It doesn't look like this covers the case where multiple org scopes are put into the same string? That seems important to cover here.

Reply:
It does, because the `testQuery` refers to two orgs: `org` and `k`, the latter via the `Repos`/`ExcludedRepos` attributes